gpa-symbols/src/ReactIcons.tsx

```diff
@@ -228,6 +228,14 @@ export namespace ReactIcons {
             <line x1="12" y1="2" x2="12" y2="15"></line>
         </svg>
 
+    export const ShareArrowDiagonal: React.FC<IProps> = (props) =>
+        <svg xmlns="http://www.w3.org/2000/svg" style={{ ...(props.Style ?? {}), width: props.Size ?? 24, height: props.Size ?? 24 }} viewBox="0 0 35 35" fill={props.Color ?? "currentColor"} stroke={props.Color ?? "currentColor"} strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" className="feather feather-share">
```

Comment:
Is there a reason we have the viewBox attribute set as `0 0 35 35`, rather than `0 0 24 24`?

Reply:
We need to have the viewbox a little larger to see the whole thing. The actual size should still be determined by the size prop

Reply:
In that case, wouldn't it be better to keep everything consistent with a viewBox of `0 0 24 24` and adjust the `ShareArrowDiagonal` to fit within that viewBox?